code/perception/src/PositionPublisher.py

```diff
@@ -107,7 +100,6 @@ def publish_current_pos(self):
         # self.loginfo("publishing data")
         temp_pos = self.update_current_pos()
         self.pos_publisher.publish(temp_pos)
-        self.heading_publisher.publish(self.current_heading)
 
```

Comment:
You could also delete L. 84 (`self.current_heading = quat2heading(..)`) and the `quat2heading`-method in the `coordinate_transformation.py` to avoid confusion

Reply:
fixed in 3edfc76